platform.md

```diff
@@ -76,18 +77,20 @@ Where:
 - `-group` MUST specify output to a `group.toml` file path as defined in the [Data Format](#data-format) section.
 - `-plan` MUST specify output to a Build Plan as defined in the [Buildpack Interface Specification](buildpack.md).
 
+#### Build phase
 To initiate the build phase, the platform MUST invoke the `/cnb/lifecycle/builder` executable with the user and environment defined in the build image config.
 Invoking this executable with no flags is equivalent to the following invocation including all accepted flags and their default values.
 
 ```bash
-/cnb/lifecycle/builder -buildpacks /cnb/by-id -group ./group.toml -plan ./plan.toml
+/cnb/lifecycle/builder -buildpacks /cnb/by-id -group ./group.toml -plan ./plan.toml -project ./project.toml
```

Comment:
We shouldn't overload the name `project.toml`, which is also proposed to be the app descriptor file.

Reply:
Also, this flag should be an argument to the exporter, not the builder.

Reply:
Maybe just leave this out for now, given that exporter is unspecified.

Reply:
We've created an Export Phase section and assumed an exporter binary to provide the flag to.  It would be good to get some feedback on this section.
We've also renamed the toml file to sources-metadata.  Not sure if that's the best name, but we're open to suggestions.